apps/meteor/app/lib/server/lib/notifyListener.ts

```diff
@@ -251,6 +263,20 @@ export const notifyOnLivechatInquiryChangedById = withDbWatcherCheck(
 	},
 );
 
+export const notifyOnLivechatInquiryChangedByVisitorIds = withDbWatcherCheck(
+	async (
+		visitorIds: ILivechatInquiryRecord['v']['_id'][],
+		clientAction: Exclude<ClientAction, 'removed'> = 'updated',
+		diff?: Partial<Record<keyof ILivechatInquiryRecord, unknown> & { queuedAt: unknown; takenAt: unknown }>,
```

Comment:
both are Dates, do we need the `unknown`?

Reply:
We're typing these 2 props as unknown everywhere in this file. Maybe @ricardogarim can tell us why :thinking: or if it's ok to provide a more specific typing